lib/floe/workflow/payload_template.rb

```diff
@@ -4,35 +4,85 @@ module Floe
   class Workflow
     class PayloadTemplate
       def initialize(payload)
-        @payload = payload
+        verify_payload!(payload)
```

Comment:
can we incorporate this in `parse_payload`?
Or do you feel it works better if you have separate?

I feel like an `else` or something in `parse` may meet our needs

Reply:
:laughing: I moved this out of there to appease codeclimate.  I actually agree it was a little simpler in `parse_payload` but that cognitive complexity was too high

Reply:
Okay you convinced me to not change code just to fix a 6/5 cyclomatic complexity